modules/rating/src/main/java/glicko2/RatingCalculator.java

```diff
@@ -260,10 +284,11 @@ private double outcomeBasedRating(Rating player, List<Result> results) {
    *
    * @param phi
    * @param sigma
+   * @param elapsedRatingPeriods
    * @return new rating deviation
    */
-  private double calculateNewRD(double phi, double sigma) {
-    return Math.sqrt( Math.pow(phi, 2) + Math.pow(sigma, 2) );
+  private double calculateNewRD(double phi, double sigma, double elapsedRatingPeriods) {
+    return Math.sqrt( Math.pow(phi, 2) + elapsedRatingPeriods * Math.pow(sigma, 2) );
```

Comment:
shouldn't `elapsedRatingPeriods` have a min floor so that the second term always contributes, even if games are played back-to-back?

Reply:
The idea is that RD should only go up based on time, so the second term not contributing is as intended.